components/sup/src/manager/commands.rs

```diff
@@ -50,6 +51,21 @@ use util;
 
 static LOGKEY: &'static str = "CMD";
 
+lazy_static! {
+    static ref RPC_CALLS: CounterVec = register_counter_vec!(
+        "hab_sup_rpc_call_total",
+        "Total number of RPC calls",
+        &["name"]
+    )
+    .unwrap();
+    static ref RPC_CALL_DURATION: HistogramVec = register_histogram_vec!(
```

Comment:
Ah, you're right.  Great catch.  I'll fix this.

Reply:
Actually, thinking about this some more, is it even possible to truly measure RPC latency?  It seems like doing that would require starting a timer on the client side, possibly around [here](https://github.com/habitat-sh/habitat/blob/master/components/sup-client/src/lib.rs#L159), where the client connects to the supervisor's control gateway, and stopping it after it receives a response.  Doesn't network latency matter most from the client's point of view?

Reply:
I would expect a measure of RPC latency to be from the client's perspective: starting the timer when issuing the network call and stopping it when receiving the response so that it includes both the network roundtrip as well as any processing time on the remote side. In many cases, I'd expect the network roundtrip to dominate the overall call time, but not for all, and those are probably the cases we care about most because they're optimization opportunities. There's not much we can do about the network, but if we're spending too much time locally executing (possibly waiting on disk I/O, or because we're CPU bound), we can make changes to address that.